### PR TITLE
WRN-3673: Migrate DayPicker unit tests to Testing Library

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated unit tests from Enzyme to Testing Library for DayPicker component.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-3673

### CODE REVIEW

File: DayPicker/tests/DayPicker-specs.js [hunk 1]
```diff
@@ -1,78 +1,79 @@
...
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
 import DayPicker from '../DayPicker';
 
 describe('DayPicker', () => {
 	describe('#aria-label', () => {
 		test(
 			'should use title, selected long string when day is single selected',
 			() => {
```
Comment (alexandrumorariu) [inline DayPicker/tests/DayPicker-specs.js]: Please put this in a single line

File: DayPicker/tests/DayPicker-specs.js [hunk 2]
```diff
@@ -1,78 +1,79 @@
...
+				const actual = screen.getByRole('group');
 
-				expect(actual).toBe(expected);
+				expect(actual).toHaveAttribute('aria-label', expected);
 			}
 		);
 
 		test(
 			'should use title, selected long string when day is multi selected',
 			() => {
```
Comment (alexandrumorariu) [inline DayPicker/tests/DayPicker-specs.js]: Same here.

File: DayPicker/tests/DayPicker-specs.js [hunk 3]
```diff
@@ -1,78 +1,79 @@
...
+
 import DayPicker from '../DayPicker';
 
 describe('DayPicker', () => {
 	describe('#aria-label', () => {
 		test(
 			'should use title, selected long string when day is single selected',
 			() => {
-				const subject = mount(
+				render(
```
Comment (alexandrumorariu) [inline DayPicker/tests/DayPicker-specs.js]: You can also write the render in a single line if there are not many props.

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Review by daniel-stoian-lgp — APPROVED** [review]

**Pull request merged**